barchart/app.js

```diff
@@ -68,57 +70,54 @@ angular.module('myApp', ['ngStorage', 'ngMaterial', 'ngMessages' ,'ngSanitize',
             $interval.cancel(promise);
             vm.wallTweets = [];
             vm.limit = 25
-            
+
             // clear form
             // $scope.query={};
-            
+
             // first call
             queryService.queryTweets(type, term, dateFrom, dateTo, maxHashtags, count);
 
             // poll every few minutes
-            promise = $interval(function(){
+            promise = $interval(function () { 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/barchart/app.js
+++ b/barchart/app.js
@@ -78,7 +78,7 @@
             queryService.queryTweets(type, term, dateFrom, dateTo, maxHashtags, count);
 
             // poll every few minutes
-            promise = $interval(function () { 
+            promise = $interval(function () {
                 queryService.queryTweets(type, term, dateFrom, dateTo, maxHashtags, count);
             } , interval*60000);
         }
```